src/web/mjs/connectors/ReaperScans.mjs

```diff
@@ -27,80 +28,71 @@ export default class ReaperScans extends Connector {
             }
         };
     }
+
     async _getMangas() {
-        let mangaList = [];
+        const mangaList = [];
         for (let page = 1, run = true; run; page++) {
+            await this.wait(this.config.throttle.value);
             let mangas = await this._getMangasFromPage(page);
             mangas.length > 0 ? mangaList.push(...mangas) : run = false;
         }
         return mangaList;
     }
+
     async _getMangasFromPage(page) {
-        let uri = new URL(this.path + '?page='+page, this.url);
-        let request = new Request(uri, this.requestOptions);
-        let data = await this.fetchDOM(request, this.queryMangas);
+        const uri = new URL(this.path + '?page='+page, this.url);
+        const request = new Request(uri, this.requestOptions);
+        await this.wait(this.config.throttle.value);
```

Comment:
this double the throttling as it's also called from getMangas.
I suggest to remove it